Header header
bool note_on
byte note_number
byte note_velocity
